package.json

```diff
@@ -77,12 +75,14 @@
     "react-dom": "^16.13.1",
     "react-is": "^16.13.1",
     "react-scripts": "3.4.1",
+    "rollup": "^2.56.3",
+    "rollup-plugin-node-resolve": "^5.2.0",
+    "rollup-plugin-rename": "^1.0.1",
+    "rollup-plugin-sass": "^1.2.6",
     "sass": "^1.32.12",
     "sass-loader": "^9.0.2",
     "tsd-jsdoc": "^2.5.0",
-    "webpack": "^5.51.1",
-    "webpack-cli": "4.8.0",
-    "@playcanvas/observer": "1.0.4"
+    "webpack": "^5.51.1"
```

Comment:
`npm uninstall webpack` or is it still required for other things?

Reply:
we still need it for storybook